packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/oktatookta/appatidp.md

```diff
@@ -1,29 +1,22 @@
-> **Note**: This guide will refer to the Okta org that represents the Identity Provider as **Org A** and the second Okta org as **Org B**.
+When you are configuring federation between two Okta orgs using OpenID Connect:
 
-1. At **Org A**, select **Applications** and then click **Add Application**.
+1. At the Okta org that represents the Identity Provider, select **Applications** and then click **Add Application**.
 
 2. Click **Create New App**.
```

Comment:
This step isn't needed. From the Developer Console, you simply select Applications and then Add Application and the wizard appears. 
<img width="1130" alt="Screen Shot 2020-12-08 at 12 40 37 PM" src="https://user-images.githubusercontent.com/45509385/101533149-b7e0da00-3952-11eb-9f6f-5116038424c1.png">

They need to then click the Web tile to move forward. 
<img width="1120" alt="Screen Shot 2020-12-08 at 12 40 21 PM" src="https://user-images.githubusercontent.com/45509385/101533171-bd3e2480-3952-11eb-92ad-5a4cc7ab9263.png">